[docs] update docs and changelog abou 64 bits and app bundles support (#4902)
expo/expo: docs/pages/versions/unversioned/distribution/building-standalone-apps.md

### If you choose to build for Android

> When building for android you can choose to build APK(`expo build:android -t apk`) or Android App Bundle (`expo build:android -t app-bundle`).

Reviewer: ```suggestion
When building for android you can choose to build APK (`expo build:android -t apk`) or Android App Bundle (`expo build:android -t app-bundle`).
```

> App bundles are recommended, but it's possible to use them only for applications that have [App Signing by Google Play](./app-signing) enabled,

Reviewer: ```suggestion
App bundles are recommended, but you have to make sure the [Google Play App Signing](./app-signing) is enabled for your project,
```
